### PR TITLE
Allow passing multiple categories

### PR DESCRIPTION
**Pull request by sourishkrout**
This PR will treat the following code snippet like a list as opposed to the very last category overwrite all preceding ones.

```$ runme run -c buzz -c bar```

### CODE REVIEW

**Comment by sourishkrout** [comment]
Review please @pastuxso.

File: internal/cmd/run.go [hunk 1]
```diff
@@ -89,16 +88,26 @@ func runCmd() *cobra.Command {
...
 						}
 
-						if category != "" {
+						if len(categories) > 0 {
 							if block.ExcludeFromRunAll() {
 								continue
 							}
 
-							if !slices.Contains(strings.Split(block.Category(), ","), category) {
+							bcats := strings.Split(block.Category(), ",")
```
Comment (adambabik) [inline internal/cmd/run.go:100]: Suggestion: Add `block.Categories()` and mark `block.Category()` as deprecated using `// Deprecated: Use Categories() instead`.

File: internal/cmd/run.go [hunk 2]
```diff
@@ -229,8 +238,8 @@ func runCmd() *cobra.Command {
 					if runMany && parallel {
 						scriptRunText = "Running"
 						blockNames = []string{blockColor.Sprint("all tasks")}
-						if category != "" {
-							blockNames = []string{blockColor.Sprintf("tasks for category %s", category)}
+						if len(categories) > 0 {
+							blockNames = []string{blockColor.Sprintf("tasks for category %s", categories)}
```
Comment (adambabik) [inline internal/cmd/run.go:242]:
```suggestion
							blockNames = []string{blockColor.Sprintf("tasks for categories %s", categories)}
```

**Review by adambabik — APPROVED** [review]

**Review by pastuxso — COMMENTED** [review]
late review, LGTM!

**Pull request merged**